saos-batch/src/test/java/pl/edu/icm/saos/batch/core/importer/CtJudgmentImportJobTest.java

```diff
@@ -0,0 +1,243 @@
+        assertThat(judgment.getTextContent(), is("22/3/A/2005\n\nWYROK\nz dnia 8 marca 2005 r.\nSygn. akt K 27/03*\n\n* Sentencja została ogłoszona dnia 15 marca 2005 r. ..."));
+        assertThat(judgment.getJudgmentType(), is(JudgmentType.SENTENCE));
+        
+        assertCtSpecificEmpty(judgment);
+    }
+    
+    private void assertCorrections_04e47013023d2b315b841f99ccb9c290_afterUpdate() {
+        ConstitutionalTribunalJudgment judgment = judgmentRepository.findOneBySourceCodeAndSourceJudgmentId(
+                SourceCode.CONSTITUTIONAL_TRIBUNAL, "04e47013023d2b315b841f99ccb9c290", ConstitutionalTribunalJudgment.class);
+        judgment = judgmentRepository.findOneAndInitialize(judgment.getId());
+        Judge correctedJudge = judgment.getJudge("Adam Jamróz");
+        
+        List<JudgmentCorrection> judgmentCorrections = judgmentCorrectionRepository.findAllByJudgmentId(judgment.getId());
+        
+        assertEquals(1, judgmentCorrections.size());
+        assertTrue(judgmentCorrections.contains(
+                createFor(judgment).update(correctedJudge).property(NAME)
+                .oldValue("sędzia Adam Jamróz").newValue("Adam Jamróz").build()));
+    }
+    
+    private void assertCtSpecificEmpty(ConstitutionalTribunalJudgment judgment) {
+        assertThat(judgment.getDecision(), is(nullValue()));
+        assertThat(judgment.getSummary(), is(nullValue()));
+        
+        assertThat(judgment.getKeywords(), is(empty()));
+        assertThat(judgment.getLegalBases(), is(empty()));
+        assertThat(judgment.getReferencedRegulations(), is(empty()));
+        
+        assertThat(judgment.getSourceInfo().getPublicationDate(), is(nullValue()));
+        assertThat(judgment.getSourceInfo().getPublisher(), is(nullValue()));
+        assertThat(judgment.getSourceInfo().getReviser(), is(nullValue()));
+
+    }
+    
+    private void assertDissentingOpinion(ConstitutionalTribunalJudgmentDissentingOpinion dissentingOpinion,
+            String textContent, String ... authors) {
+        assertThat(dissentingOpinion.getTextContent(), is(textContent));
+        assertThat(dissentingOpinion.getAuthors(), containsInAnyOrder(authors));
+    }
+    
+    private void assertJudge(Judgment judgment, String name, JudgeRole... expectedRoles) {
+        Judge judge = judgment.getJudge(name);
+        assertNotNull(judge);
+        assertThat(judge.getSpecialRoles(), Matchers.containsInAnyOrder(expectedRoles));
+    }
+    
+    private void assertSourceJudgmentIds(String... sourceJudgmentIds) {
+        List<String> ids = judgmentRepository.findAll().stream()
+                .filter(j -> j.getCourtType() == CourtType.CONSTITUTIONAL_TRIBUNAL)
+                .map(j -> j.getSourceInfo().getSourceJudgmentId())
+                .collect(Collectors.toList());
+        assertThat(ids, containsInAnyOrder(sourceJudgmentIds));
+    }
+    
+    private void resetImportDir(String importDirClasspath) {
```

Comment:
The method is also in ScJudgmentImportJobTest. It is not very wise to have the same methods in multiple classes. Let's move it to a package private helper class e.g. JudmentJobTestHelper